pubsub/cloud-client/subscriber_test.py

```diff
@@ -169,13 +169,15 @@ def test_update(subscriber_client, subscription, capsys):
 def _publish_messages(publisher_client, topic):
     for n in range(5):
         data = u'Message {}'.format(n).encode('utf-8')
-        publisher_client.publish(
+        future = publisher_client.publish(
             topic, data=data)
+        future.result()
```

Comment:
Do we care about printing the result of this?

Reply:
Same as above.